docs/UserGuide.md

```diff
@@ -68,6 +72,48 @@ An image of our UI is shown below!
 
 ---
 
+## Structure of the User Guide
+
+Worries about how tedious it will be to read this document? Not to worry! We have structured this User Guide in a way
+that makes it easy and quick for you to find what you need. In this next subsection,
+[Reading this User Guide](#reading-this-user-guide), you can find some tips we have on how to read this guide. The
+next section, [Features](#features), documents the main functionalities of _Teaching Assistant_ and how to use them.
+
+The main functions of _Teaching Assistant_ can be summmarised as the following:
```

Comment:
Not sure if this part is needed, would it be repeating what's already in the intro?

Reply:
can re-emphasise i guess

Reply:
can re-emphasise haha

Reply:
okies